quick-integration.hcl

```diff
@@ -0,0 +1,58 @@
+param "converge-bin-dir" {}
+
+param "image-name" {
+  default = "ubuntu"
+}
+
+param "image-tag" {
+  default = "latest"
+}
+
+docker.container "converge" {
```

Comment:
you should use the docker.image resource to make sure that the image is present. this fails on a clean machine without that.